int32 no_obstacle
float32[] sizes_of_obstacle
coords[] obs_coord 
obs_range[] ranges
obs_theta[] theta
int32 smallest_obs
int32 next_obs
